qcengine/programs/entos.py

```diff
@@ -124,14 +138,14 @@ def compute(self, input_data: "AtomicInput", config: "JobConfig") -> "AtomicResu
             return UnknownError(proc["stderr"])
 
     def execute(
-        self,
-        inputs: Dict[str, Any],
-        extra_infiles: Optional[Dict[str, str]] = None,
-        extra_outfiles: Optional[List[str]] = None,
-        extra_commands: Optional[List[str]] = None,
-        scratch_name: Optional[str] = None,
-        scratch_messy: bool = False,
-        timeout: Optional[int] = None,
+            self,
+            inputs: Dict[str, Any],
+            extra_infiles: Optional[Dict[str, str]] = None,
+            extra_outfiles: Optional[List[str]] = None,
+            extra_commands: Optional[List[str]] = None,
+            scratch_name: Optional[str] = None,
+            scratch_messy: bool = False,
```

Comment:
Odd that black is changing this.

Reply:
Sorry I don't think black changed that. I believe that was the built in PEP8 enforcer from PyCharm. Should I undo the indent change?

Reply:
Yes, we use `black` now to format everything. Looks like PyCharm can integrate: https://black.readthedocs.io/en/stable/editor_integration.html

Reply:
Thanks! Just reformatted using black.